### ISSUE TITLE
Over wan is not working

### ISSUE THREAD

**Issue by bhati6734** [qH0sT/AndroSpy#29]
Hallo  i am using portmap.io
And open vpn for portforwarding 
But the software is not communicating with rat trojan

And the same portmap.io is working with 888 rat witbout any probpem

**Comment by qH0sT** [comment]
> Hallo i am using portmap.io
> And open vpn for portforwarding
> But the software is not communicating with rat trojan
> 
> And the same portmap.io is working with 888 rat witbout any probpem

When I try with DuckDns it is working well, I don't know why it is not working with open vpn.

**Comment by bhati6734** [comment]
Have you tested it on portmap.io

**Comment by qH0sT** [comment]
> Have you tested it on portmap.io

No, I did not test but I use my own RAT with duckdns very well. I don't know problem.

**Comment by bhati6734** [comment]
So , can you share a video how to hack over wan
I have duckdns ip

**Comment by qH0sT** [comment]
> So , can you share a video how to hack over wan
> I have duckdns ip

I can't open port because I am in the CGN pool but one user hack over WAN:

https://user-images.githubusercontent.com/19536819/101899315-70b34d00-3bcf-11eb-859e-8dc36421601b.png

**Comment by bhati6734** [comment]
Sir your the only person who can help its user 
To hack over wan

Using openvpn portmap.io

Portmap.io is the most used website by users for port forwarding

Please share your telegram id 
So i can get help directly from you

**Comment by qH0sT** [comment]
> Sir your the only person who can help its user
> To hack over wan
> 
> Using openvpn portmap.io
> 
> Portmap.io is the most used website by users for port forwarding
> 
> Please share your telegram id
> So i can get help directly from you
> 
> And one more thing is to add port support from 1-99999
> 
> If someone have 443 port your rat did not support thats

ok, I will try with portmap.io

**Comment by qH0sT** [comment]
@bhati6734 wtf, Open VPN can't connect to portmap.io. I can't try it with portmap.io.

**Comment by bhati6734** [comment]
In my knowledge every 4 person out of 5 
Using portmap.io for portforwarding
And i am using portmap.io in 888rat spymax etc..

If you can't bero

Please suggest us how it will work over wan

**Comment by qH0sT** [comment]
> In my knowledge every 4 person out of 5
> Using portmap.io for portforwarding
> And i am using portmap.io in 888rat spymax etc..
> 
> If you can't bero
> 
> Please suggest us how it will work over wan

I can't connect to portmap.io via Open VPN. If I could connect I would try but I will retry again.

**Comment by qH0sT** [comment]
@bhati6734 
![1](https://user-images.githubusercontent.com/45147475/102017324-5fa54000-3d77-11eb-975f-6fe19a40fc65.PNG)

**Comment by bhati6734** [comment]
Brother see this video 
Portforwarding with portmap.io
On spynote rat hack over wan
https://youtu.be/qlLWu9yVMKM

**Comment by BurntDog** [comment]
portmap.io works for myself, live mic does not and live stream/snap shot very laggy key logger also a no go, yes it was permitted in settings.

@qH0sT I had a issue connecting to portmap.io via Open VPN
This fixed my issue:
Reset Network TCP/IP stack
If your TCP/IP stack is corrupted it can be reseted with the following commands.

Windows XP:

Search for Command Prompt > Run As Administrator
netsh winsock reset
netsh int ip reset
ipconfig /flushdns
Restart computer
Windows Vista, 7, 8, 10:

Search for Command Prompt > Run As Administrator
netsh winsock reset catalog
netsh int ipv4 reset reset.log
netsh int ipv6 reset reset.log
Restart computer

**Comment by BurntDog** [comment]
Reset Network TCP/IP stack
If your TCP/IP stack is corrupted it can be reseted with the following commands.

Windows XP:

Search for Command Prompt > Run As Administrator
netsh winsock reset
netsh int ip reset
ipconfig /flushdns
Restart computer
Windows Vista, 7, 8, 10:

Search for Command Prompt > Run As Administrator
netsh winsock reset catalog
netsh int ipv4 reset reset.log
netsh int ipv6 reset reset.log
Restart computer

**Comment by BurntDog** [comment]
> Hallo i am using portmap.io
> And open vpn for portforwarding
> But the software is not communicating with rat trojan
> 
> And the same portmap.io is working with 888 rat witbout any probpem

Reset Network TCP/IP stack
If your TCP/IP stack is corrupted it can be reseted with the following commands.

Windows XP:

Search for Command Prompt > Run As Administrator
netsh winsock reset
netsh int ip reset
ipconfig /flushdns
Restart computer
Windows Vista, 7, 8, 10:

Search for Command Prompt > Run As Administrator
netsh winsock reset catalog
netsh int ipv4 reset reset.log
netsh int ipv6 reset reset.log
Restart computer
